Description: Change default backend to theano
 Change default backend to theano.
Index: keras/keras/backend/load_backend.py
===================================================================
--- keras.orig/keras/backend/load_backend.py
+++ keras/keras/backend/load_backend.py
@@ -24,8 +24,8 @@ else:
         _keras_base_dir = '/tmp'
     _keras_dir = os.path.join(_keras_base_dir, '.keras')
 
-# Default backend: TensorFlow.
-_BACKEND = 'tensorflow'
+# Default backend: Theano (Debian).
+_BACKEND = 'theano'
 
 # Attempt to read Keras config file.
 _config_path = os.path.expanduser(os.path.join(_keras_dir, 'keras.json'))
